<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmScalar</class>
 <widget class="QMainWindow" name="frmScalar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>349</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>云图颜色映射</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabScaleRange">
     <attribute name="title">
      <string>颜色范围</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>341</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>线性映射</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>最大值</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>最小值</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtMax">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>150</y>
         <width>171</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtMin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>171</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbAuto">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>401</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>自动填充</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbFixed">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>401</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>固定值</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>141</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>颜色数</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtNumOfColors">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>190</y>
         <width>171</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
